docs/documentation/updating-the-kit.md

```diff
@@ -2,39 +2,41 @@
 
 1. [Download the zip file of the latest version of the Prototype Kit](/docs/download).
 
-2. Unzip the zip file.
+2. Unzip the zip file. It will make a folder called `govuk-prototype-kit`, with a version number.
 
 3. Make a backup copy of your prototype folder.
 
-4. In your prototype, delete everything except the `app` and `.git` folders.
+4. You'll need to copy files that start with a . (‘dot files’). Operating systems hide dot files by default. You can:
```

Comment:
This might be a clearer explanation?

```suggestion
4. You'll need to copy some files that operating systems do not usually show. These hidden files usually start with a period `.` and are known as ‘dot files’.

To show files, you can:
```

Reply:
thanks! have made the change but dropped 'period' as not sure about that vs full-stop